modules/manage/pages/cluster-maintenance/compaction-settings.adoc

```diff
@@ -26,6 +26,56 @@ Compaction policy may be applied to a cluster or to an individual topic. If both
 * `compact`: This triggers only cleanup of messages with multiple versions. A message that represents the only version for a given key is not deleted.
 * `compact,delete`: This combines both policies, deleting messages exceeding the retention period while compacting multiple versions of messages.
 
+== Tombstone record removal
+
+When a topic's messages should be deleted, for example as data is deleted from a source system, clients add a tombstone record to the message log. A tombstone contains the message key, and the value `null`. Tombstones signal to brokers and consumers that records with the same key prior to it in the log should be deleted. 
```

Comment:
> When a topic's messages should be deleted

"Messages" and "records" are used synonymously in redpanda docs, but is there a preferred term? I.e [rpk topic consume](https://docs.redpanda.com/redpanda-cloud/reference/rpk/rpk-topic/rpk-topic-consume/) mostly refers to "records" (including when it refers to tombstones)

Also, the way this first sentence is phrased makes it sound like a single tombstone record can be responsible for the deletion of many "messages" in a topic. Would rephrase to something like 

> Compaction can also enable the deletion of existing records through the use of tombstones. For example, as data is deleted from a source system, clients produce a tombstone record to the message log. A tombstone contains a key and the value `null`. Tombstones signal to brokers and consumers that records with the same key prior to it in the log should be deleted.